postgres/CHANGELOG.md

```diff
@@ -1,5 +1,10 @@
 # CHANGELOG - postgres
 
+2.1.1 / 2018-03-13
```

Comment:
can you put the string `Unreleased` instead of the data? So that our release script will swap it with the actual release date